### ISSUE TITLE
Netfabb has discontinued Multi-Gantry feature

### ISSUE THREAD

**Issue by mohin009** [Autodesk/machine-collaboration-utility#47]
Hello, Netfabb has discontinued the multi-gantry feature in the latest version. Is there any other way to proceed with multi-gantry with MCU?

Ref: https://forums.autodesk.com/t5/netfabb-forum/multi-gantry/m-p/9451920

**Comment by edwin3d3d** [comment]
Hello there,

     Looks like they might not have completely. I have just paid for a 1yr subscription and I downloaded the 2020 version. I did find that I can download all of the versions back to 2018. It looks as if it is functioning in the 2018 version and maybe the 2019 version, tho I haven't check that yet. I have been playing with it all evening and I have been able to create a profile for a machine I am contemplating building ( generic Multi gantry FFF Machine) and I even was able to generate tool paths and end up with the required .esh file. Right now I am experimenting further with the MCU utility on a raspberry Pi and i'm going to see if there is any progress to be made there. Feel free to touch base if you want to collaborate.

**Comment by innovativemann** [comment]
@edwin3d3d  look like you are doing the same thing aa I am I would love to collaborate with you and @mohin009  is my teammate  drop me your contacts so we can talk, please

**Comment by edwin3d3d** [comment]
Yes! We should connect. Maybe we can make some progress. I'm finding it just out side my wheel house of skills. There needs to be a independent project to get this popularized. I'm away from my home for a few more weeks so sadly no progress at this time. Looking forward
